src/preferencesControl.ts

```diff
@@ -142,6 +143,7 @@ export class PreferencesControl {
     public showPreferencesDialog(): void {
         let modal: HTMLElement = <HTMLElement> document.getElementsByClassName(styles.cookieModal)[0];
         if (modal) {
+            this.previousFocusElementBeforePopup = <HTMLElement> document.activeElement;
```

Comment:
Can you set `previousFocusElementBeforePopup` back to null when dialog is closed?